src/es_archiver/lib/archives/__tests__/format.ts

```diff
@@ -43,11 +43,11 @@ describe('esArchiver createFormatArchiveStreams', () => {
     });
 
     it('streams consume js values and produces buffers', async () => {
-      const output = await createPromiseFromStreams([
+      const output = await createPromiseFromStreams<Buffer[]>([
```

Comment:
Our internal stream library miss proper TS typing, the types are not properly coerced from the transform chain types. But converting to RX is way out of the scope of current PR.